### PR TITLE
Add azure-mgmt-communication track 2

### PR DESCRIPTION
**Pull request by memontic-ms**

### CODE REVIEW

File: sdk/communication/azure-mgmt-communication/sdk_packaging.toml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 [packaging]
-auto_update = false
 package_name = "azure-mgmt-communication"
-package_pprint_name = "Communication Management"
-package_doc_id = "communication"
+package_nspkg = "azure-mgmt-nspkg"
```
Comment (usvoyager) [inline sdk/communication/azure-mgmt-communication/sdk_packaging.toml:3]: >package_nspkg [](start = 0, length = 13)

is this right?

File: sdk/communication/azure-mgmt-communication/README.md [hunk 2]
```diff
@@ -1,75 +1,21 @@
 # Microsoft Azure SDK for Python
 
-This is the Microsoft Azure Communication Management Client Library.
+This is the Microsoft Azure MyService Management Client Library.
```
Comment (usvoyager) [inline sdk/communication/azure-mgmt-communication/README.md:3]: >This [](start = 0, length = 4)

revert

File: sdk/communication/azure-mgmt-communication/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 1.0.0b1 (2020-09-22)
+## 0.1.0 (1970-01-01)
```
Comment (mitchdenny) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:3]: Doesn't seem right.

Comment (usvoyager) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:3]: >## [](start = 0, length = 2)

revert and update

Comment (changlong-liu) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:3]: this should be align with version in _version.py

Comment (memontic-ms) [reply sdk/communication/azure-mgmt-communication/CHANGELOG.md:3]: Should we keep the 1.0.0b1 (2020-09-22) in the history? As in, have two entries here.

File: sdk/communication/azure-mgmt-communication/sdk_packaging.toml (hunk 1 above)
Comment (memontic-ms) [reply sdk/communication/azure-mgmt-communication/sdk_packaging.toml:3]: Updating all MyService to Communication

File: sdk/communication/azure-mgmt-communication/README.md (hunk 2 above)
Comment (memontic-ms) [reply sdk/communication/azure-mgmt-communication/README.md:3]: Reverting

File: sdk/communication/azure-mgmt-communication/sdk_packaging.toml [hunk 4]
```diff
@@ -1,6 +1,8 @@
 [packaging]
-auto_update = false
 package_name = "azure-mgmt-communication"
+package_nspkg = "azure-mgmt-nspkg"
 package_pprint_name = "Communication Management"
-package_doc_id = "communication"
+package_doc_id = ""
```
Comment (usvoyager) [inline sdk/communication/azure-mgmt-communication/sdk_packaging.toml:5]: >"" [](start = 17, length = 2)

"communication"

Comment (memontic-ms) [inline sdk/communication/azure-mgmt-communication/sdk_packaging.toml:5]: Should package_doc_id field be filled?

Comment (memontic-ms) [reply sdk/communication/azure-mgmt-communication/sdk_packaging.toml:5]: Fine as-is. Field not needed

File: sdk/communication/azure-mgmt-communication/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0b2 (2020-09-22)
+
+* Updates for track 2
```
Comment (jsntcy) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:5]: The release note makes no sense to SDK users. Please refer to https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/resources/azure-mgmt-resource/CHANGELOG.md#1500b1-2020-06-17 which transit from track1 to track2.

Comment (memontic-ms) [reply sdk/communication/azure-mgmt-communication/CHANGELOG.md]: Changelog has been updated with additional information

**Review by usvoyager — APPROVED** [review]
:shipit:

File: sdk/communication/azure-mgmt-communication/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
 
+## 1.0.0b2 (2020-09-22)
+
+**General breaking changes**
+
+Learn more about migrating to the next generation of the Azure Python SDK in the [migration guide for resource management](https://github.com/Azure/azure-sdk-for-python/blob/master/doc/sphinx/python_mgmt_migration_guide.rst)
+
+- Credential system has been completly revamped:
+
+  - `azure.common.credentials` or `msrestazure.azure_active_directory` instances are no longer supported, use the `azure-identity` classes instead: https://pypi.org/project/azure-identity/
```
Comment (jsntcy) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:11]: It seems you miss other breaking changes https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/resources/azure-mgmt-resource/CHANGELOG.md#1500b1-2020-06-17

**Review by changlong-liu — APPROVED** [review]

**Pull request merged**